src/main/java/manufacturing/dao/jdbc/DriverDaoJdbcImpl.java

```diff
@@ -0,0 +1,122 @@
+package manufacturing.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import manufacturing.dao.DriverDao;
+import manufacturing.exception.DataProcessingException;
+import manufacturing.lib.Dao;
+import manufacturing.model.Driver;
+import manufacturing.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoJdbcImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String query = "INSERT INTO drivers "
+                + "(driver_name, license_number) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1, driver.getName());
+            preparedStatement.setString(2, driver.getLicenceNumber());
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
```

Comment:
same here with `GENERATED_KEY`, it just makes it a little more readable IMHO.

Reply:
done